test/esm/index.test.js

```diff
@@ -3,20 +3,6 @@
 const t = require('tap')
 const semver = require('semver')
 
-if (semver.lt(process.versions.node, '13.3.0')) {
-  t.skip('Skip esm because Node version <= 13.3.0')
-} else {
-  // Node v8 throw a `SyntaxError: Unexpected token import`
-  // even if this branch is never touch in the code,
-  // by using `eval` we can avoid this issue.
-  // eslint-disable-next-line
-  new Function('module', 'return import(module)')('./esm.mjs').catch((err) => {
-    process.nextTick(() => {
-      throw err
-    })
-  })
-}
-
 if (semver.lt(process.versions.node, '14.13.0')) {
```

Comment:
If we want to support only LTS versions of node.js 14 then we can omit this check as well, wdyt?

Reply:
Since we're supporting Node 14, I think this test should exist. But technically speaking, this test will never run, since we're only testing LTS on CI. I'm fine with both options.